nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/settings-content.jsp

```diff
@@ -77,7 +77,7 @@
             Last updated:&nbsp;<span id="settings-last-refreshed" class="value-color"></span>
         </div>
         <div id="settings-loading-container" class="loading-container"></div>
-        <div id="controller-cs-availability" class="hidden">Listed services are available to all Reporting Tasks and services defined in the Controller Settings.</div>
+        <div id="controller-cs-availability" class="hidden">Listed services are available to all Reporting Tasks, Parameter Providers, and services defined in the Controller Settings.</div>
```

Comment:
The services are also available to Registry Clients. Can you note that here too?

Reply:
@mcgilman Thanks for reviewing! I've added it.